libcflib/schemas.py

```diff
@@ -118,6 +118,7 @@
                 "type": "dict",
             },
             "name": {"schema": {"type": "string"}, "type": "list"},
+            "metdata_version": {"schema": {"type": "integer"}, "type": "integer"},
```

Comment:
I don't think you need the `schema` chunk here, that is mostly for dict and list types since they have to have types for the data.

Reply:
so is this just int?